src/components/AssetManagementPanel/AssetManagementForm/AssetManagementTitle/AssetManagementTitle.tsx

```diff
@@ -15,7 +15,7 @@ export const AssetManagementTitle = styled(({ className, onBack, formAction, dis
   return (
     <div className={`row ${className}`}>
       <div className="col-12">
-        <div className="action-back" onClick={onBack} data-disabled={disabled}>
+        <div className="action-back" onClick={() => onBack(disabled)} data-disabled={disabled}>
```

Comment:
case in point, *child already know it is disabled*, so can let it fire onBack only if it's not disabled.

Reply:
updated the component.